WardenApi/src/main/java/com/salesforce/dva/warden/dto/Policy.java

```diff
@@ -1,5 +1,164 @@
 package com.salesforce.dva.warden.dto;
 
-public class Policy {
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * Policy Dto.
+ *
+ * @author  Jigna Bhatt (jbhatt@salesforce.com)
+ */
+@SuppressWarnings("serial")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Policy extends com.salesforce.dva.warden.dto.Entity {
+	//~ Instance fields ******************************************************************************************************************************
+
+	    private String servcie;		
+	    private String name;
+	    private List<String> owner  = new ArrayList<String>();
+	    private List<String> user  = new ArrayList<String>();
+	    private String subSystem;
+	    private String metricName;
+	    private String triggerType;
+	    private String aggregator;
+	    private List<Double> threshold;	    
+	    private String timeUnit;
+	    private double defaultValue;
+	    private String cronEntry;
+	    private List<BigInteger> suspensionLevels = new ArrayList<BigInteger>();
+
+		public String getServcie() {
+			return servcie;
+		}
+
+		public void setService(String service) {
+			this.servcie = service;
+		}
+
+		public String getName() {
+			return name;
+		}
+
+		public void setName(String name) {
+			this.name = name;
+		}
+
+		public List<String> getOwner() {
+			return owner;
+		}
+
+		public void setOwner(List<String> owner) {
+			this.owner = owner;
+		}
+
+		public List<String> getUser() {
+			return user;
+		}
+
+		public void setUser(List<String> user) {
+			this.user = user;
+		}
+
+		public String getSubSystem() {
+			return subSystem;
+		}
+
+		public void setSubSystem(String subSystem) {
+			this.subSystem = subSystem;
+		}
+
+		public String getMetricName() {
+			return metricName;
+		}
+
+		public void setMetricName(String metricName) {
+			this.metricName = metricName;
+		}
+
+		public String getTriggerType() {
+			return triggerType;
+		}
+
+		public void setTriggerType(String triggerType) {
```

Comment:
Fields that correspond to enumerations, should restrict the values to the valid enumerated values.  For triggerType, the data type should be an enum that's added to the DTO.

public enum TriggerType {
  EQUAL,
  NOT_EQUAL,
  ...
}